client/me/pending-payments/index.jsx

```diff
@@ -124,29 +124,11 @@ PendingPayments.propTypes = {
 export default connect(
 	state => {
 		const userId = getCurrentUserId( state );
-
 		const response = getHttpData( requestId( userId ) );
```

Comment:
Not part of this diff, but `requestId()` would be better named something like `requestUrlFromUserId()`